src/Group.cpp

```diff
@@ -167,9 +167,9 @@ bool Group::isMember(Channel *curChan, Channel *aclChan, QString name, ServerUse
 		switch (args.count()) {
 			default:
 			case 3:
-				maxdesc = args[2].isEmpty() ? maxdesc : args[2].toInt();
+				maxdesc = args[2].isEmpty() ? maxdesc : args[2].toInt(); // Intentional fallthrough
```

Comment:
I don't like it on the same line.
We're commenting non-statement control flow, not this line of code.

Reply:
Right, I'm going to move it in a new line. Do you think we should wrap the code in brackets as well?

Reply:
Looks much better!